wpiutil/src/main/native/thirdparty/llvm/cpp/llvm/Windows/WindowsSupport.h

```diff
@@ -92,10 +92,10 @@ wpi::VersionTuple GetWindowsOSVersion();
 bool MakeErrMsg(std::string *ErrMsg, const std::string &prefix);
 
 // Include GetLastError() in a fatal error message.
-LLVM_ATTRIBUTE_NORETURN inline void ReportLastErrorFatal(const char *Msg) {
+[[noreturn]] inline void ReportLastErrorFatal(const char *Msg) {
   std::string ErrMsg;
   MakeErrMsg(&ErrMsg, Msg);
-  wpi::report_fatal_error(ErrMsg);
+  wpi::report_fatal_error(Twine(ErrMsg));
```

Comment:
I don’t understand this change.  It should just pass directly since there’s a const char* taking version of report_fatal_error.